libs/sdk-ui-kit/src/Dialog/DialogList/DialogListHeader.tsx

```diff
@@ -0,0 +1,76 @@
+// (C) 2022 GoodData Corporation
+
+import React, { useCallback } from "react";
+import cx from "classnames";
+
+import { Typography } from "../../Typography";
+import { Bubble, BubbleHoverTrigger } from "../../Bubble";
+
+const BUTTON_TOOLTIP_ALIGN_POINTS = [
+    { align: "cr cl", offset: { x: 0, y: 0 } },
+    { align: "cl cr", offset: { x: 0, y: 0 } },
+];
+
+/**
+ * @internal
+ */
+export interface IDialogListHeaderProps {
+    className?: string;
+    gdIconName?: string;
+    title?: string;
+    buttonTitle?: string;
+    buttonDisabled?: boolean;
+    buttonTooltipText?: string;
+    onButtonClick?: () => void;
+}
+
+/**
+ * @internal
+ */
+export const DialogListHeader: React.FC<IDialogListHeaderProps> = (props) => {
```

Comment:
Use React.VFC since this component do not have children

Reply:
Updated